src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTableIntegrationTest.java

```diff
@@ -236,4 +237,31 @@ public void testMoltenOutputWithMultipleAlleles() throws IOException {
         spec.setTrimWhiteSpace(false);
         spec.executeTest("testMoltenOutputWithMultipleAlleles", this);
     }
+
+    @Test
+    public void testNoFieldsSpecified() throws IOException {
+        final File inputFile = new File(getToolTestDataDir(), "extraheaderlinesdeleted_dbsnp_138.snippet.vcf");
+        final File outputFile = createTempFile(getToolTestDataDir(), "noFieldsSpecifiedOutput.table");
```

Comment:
The first argument to `createTempFile()` is supposed to be a prefix to the file name, and the second argument should be the file extension. So, eg., in this case you want something like:
```
final File outputFile = createTempFile("noFieldsSpecifiedOutput", ".table");
```
The temp file creation code will take care of appending a unique string to the name so that it doesn't collide with other temp files.

This same comment also applies to the `outputFile ` in `testNoFieldsSpecifiedWithSamples()` below.